### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Mmduh-483**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,115 @@
-# ib-kubernetes
+[![Build Status](https://travis-ci.com/Mmduh-483/ib-kubernetes.svg?branch=master)](https://travis-ci.com/Mmduh-483/ib-kubernetes) [![Coverage Status]
```
Comment (moshe010) [inline README.md]: what is this?

File: README.md [hunk 2]
```diff
@@ -1 +1,115 @@
...
+      * [Build](#build)
+         * [Building InfiniBand Kubernetes Binary] (#building-infiniband-kubernetes-binary)
+         * [Building Network Manager Plugins] (#building-network-manager-plugins)
+         * [Building Container Image] (#building-container-image)
+      * [Configuration reference](#configuration-reference)
+      * [Deploying](#deploying)
+
+# IB-Kubernetes
+
+InfiniBand Kubernetes listens for Kubernets pod events. It Generates guid(s) and add them to PKey with network manager
```
Comment (moshe010) [inline README.md]: what do you mean network manager

File: README.md [hunk 3]
```diff
@@ -1 +1,115 @@
...
+         * [Building InfiniBand Kubernetes Binary] (#building-infiniband-kubernetes-binary)
+         * [Building Network Manager Plugins] (#building-network-manager-plugins)
+         * [Building Container Image] (#building-container-image)
+      * [Configuration reference](#configuration-reference)
+      * [Deploying](#deploying)
+
+# IB-Kubernetes
+
+InfiniBand Kubernetes listens for Kubernets pod events. It Generates guid(s) and add them to PKey with network manager 
+for newly created pods with annotation `mellanox.infiniband.app: required` then change the value to `configured` if
```
Comment (moshe010) [inline README.md]: This implementation detail it not important

File: README.md [hunk 4]
```diff
@@ -1 +1,115 @@
...
+InfiniBand Kubernetes listens for Kubernets pod events. It Generates guid(s) and add them to PKey with network manager 
+for newly created pods with annotation `mellanox.infiniband.app: required` then change the value to `configured` if 
+passed without errors, and for removed pods with `mellanox.infiniband.app: configurd` it release the guid and remove 
+it from the pkey with the network manager. 
+
+InfiniBand Kubernetes cooperate with [Mellanox InfiniBand SR-IOV CNI](https://github.com/Mellanox/ib-sriov-cni) and 
+[Intel Multus CNI](https://github.com/intel/multus-cni), where the InfiniBand Kubernetes read the InfiniBand SR-IOV CNI 
+network attachment configuration created for Multus CNI and read the PKey, to add the newly generated Guids to that PKey.
+
+## Network Manager Plugins
```
Comment (moshe010) [inline README.md]: you mean subnet manager?

File: README.md (hunk 1 above)
Comment (Mmduh-483) [reply README.md]: Badges for Travis-CI job and Coverage.

File: README.md (hunk 2 above)
Comment (Mmduh-483) [reply README.md]: Subnet manager.

File: README.md (hunk 4 above)
Comment (Mmduh-483) [reply README.md]: yes, @moshe010 do you prefer to change it to subnet manager?

File: README.md (hunk 3 above)
Comment (Mmduh-483) [reply README.md]: removed

File: README.md (hunk 4 above)
Comment (moshe010) [reply README.md]: Yes

File: README.md (hunk 1 above)
Comment (moshe010) [reply README.md]: You need that in readme

Comment (Mmduh-483) [reply README.md]: It is not necessary, it is just to show the coverage and build status.

Comment (moshe010) [reply README.md]: ok it nice let keep it and add to the others

**Pull request merged**